pyroscope/session.go

```diff
@@ -3,7 +3,7 @@ package pyroscope
 import (
 	"bytes"
 	"github.com/pyroscope-io/client/internal/alignedticker"
-	"github.com/pyroscope-io/client/internal/cumulativepprof"
+	"github.com/pyroscope-io/godeltaprof"
```

Comment:
This is a major change. I wonder if we should do a major release or, maybe, make this logic configurable

Reply:
I'd go with a major release the same as we did with cumulativepprof previously

Reply:
I meant a major release that changes the import path: https://go.dev/doc/modules/major-version – the downside is that users may not migrate to the new version and we would need to maintain several major versions.

Well, technically our major version is 0, so we're allowed to break compatibility (this also relates to the configuration option we want to remove). The question is whether we are okay with switching all users to `godeltaprof` automatically – my answer is yes, after thorough testing in our own environment, I think this is fine

Reply:
agree